index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Playfair+Display&display=swap"
+      rel="stylesheet"
+    />
+    <title>Portifolio</title>
+  </head>
+  <body>
+    <header>
+      <img src="img/image_geometry_1.jpg" alt="square" class="geometry" />
+      <nav>
+        <img src="img/ic_menu@2x.jpg" alt="menu" class="menu-button" />
+      </nav>
+    </header>
+    <section class="headline">
+      <h1>Meqdam<br />Al-qudah</h1>
+
+      <p class="blockquote">
+        Hello! I am a full-stack web developer! I can help you build a product,
+        feature or website. Taka a look of my works. If you like what you see
+        and have a project you need coded, don’t hesitate and contact me.
+      </p>
+      <button>Start collaboration</button>
```

Comment:
- [ ] As requested by the previous code reviewer, which is a requirement for this project, please kindly fix the margins of the ***hamburger button***, ***blockquote***, and ***start collaboration button*** to match the design. See images below;
***Your implementation at width: 400px***
![meqI](https://user-images.githubusercontent.com/65852621/138246178-64329160-2627-405f-9558-135bc44167c6.png)

***The design***
![meqD](https://user-images.githubusercontent.com/65852621/138246348-dda156ca-25b5-463b-96d1-c0f77880111a.png)